types/base_contract.ts

```diff
@@ -14,14 +14,14 @@ export class BaseContract {
     public abi: any[];
     // public abi: Web3.AbiDefinition[];
 
-    public web3ContractInstance: Web3.ContractInstance;
+    public web3ContractInstance: any;
```

Comment:
I get really nervous when we're moving to an `any` type. Is there another type we could use?

Reply:
If it doesn't exist, we should consider creating our own type

Reply:
There is no other type right now defined at least.  `web3.eth.Contract` is a class that has a constructor to generate classes, but I couldn't get it working as a type yet

Reply:
i guess same thing should be done for the `// public abi: Web3.AbiDefinition[];` comment we have above. not sure why it was commented to be `any` there as well